Makefile

```diff
@@ -23,3 +23,7 @@ integration-test: build
 
 integration-test-py3: build-py3
 	docker run -v `$(HOST_TMPDIR)`:/tmp -v /var/run/docker.sock:/var/run/docker.sock docker-py3 py.test -rxs tests/integration_test.py
+
+integration-ci:
+	docker build -t dpy-tests -f ./tests/Dockerfile .
+	docker run --privileged -t dpy-tests
```

Comment:
Yeah, it also means you can run the tests against any arbitrary daemon by passing in the DOCKER_HOST environment variable (so down the line we can e.g. run it against a Swarm cluster).

It's also nice to have the Docker-in-Docker stuff isolated to a container that's not getting messed with.